model/YoutubeVideo.js

```diff
@@ -6,6 +6,7 @@ module.exports = YoutubeVideo = function(url, info) {
     this.info = info;
 };
 
+// Ew stop using prototypes
```

Comment:
It was more of a joking comment than anything, but I'm pretty sure you can replace all of that with the class syntax in ES6.